django/applications/catmaid/control/circles.py

```diff
@@ -72,7 +73,7 @@ def circles_of_hell(request, project_id=None):
         raise Exception("No skeletons were provided.")
 
     cursor = connection.cursor()
-    mins, relations = _clean_mins(request, cursor, int(project_id))
+    mins, relations = _clean_mins(request, cursor, int(project_id)) # FIXME: I think this will fail if project_id is None
```

Comment:
Yes, this will fail if `project_id` is `None` and we should remove the default parameter for `project_id`. Having no project is not expected, because the permission check in the `requires_user_role`  decorator needs a `project_id` as well. Without it, the code would already fail there.